MapboxSearch/place-autocomplete/src/main/java/com/mapbox/search/autocomplete/PlaceAutocompleteSuggestion.kt

```diff
@@ -88,6 +95,7 @@ public class PlaceAutocompleteSuggestion internal constructor(
     override fun hashCode(): Int {
         var result = name.hashCode()
         result = 31 * result + (formattedAddress?.hashCode() ?: 0)
+        result = 31 * result + coordinate.hashCode()
```

Comment:
can't `coordinate` be null here?

Reply:
Good catch! It'll be null in most cases. Let me fix that real quick.

Reply:
Fixed the same in both AddressAutofill and PlaceAutocomplete